src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -301,6 +305,46 @@ static ConfigDef newConfigDef() {
             6,
             ConfigDef.Width.NONE,
             SNOWFLAKE_ROLE)
+        .define(
+            AUTHENTICATOR,
+            Type.STRING,
+            "",
+            Importance.LOW,
```

Comment:
any reason this is low and rest are high importance?

Reply:
I thought this is an optional parameter when we use jwt. While `client_id` , `client_secret` and `refresh_token` is required when using oauth. Do you think we should make all of the low?